finalPrep/modZeroHero.js

```diff
@@ -48,24 +77,81 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //assessSituation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 //assessSituation(3, announcement, excuse) > should print - "Meh. Hard pass."
 
+function assessSituation2 (dangerLevel, announcement, excuse) {
+  if (dangerLevel >= 50){
+    console.log(excuse);
+  }
+  else if (dangerLevel < 10){
+    console.log('Meh. Hard pass.');
+  }
+  else {
+    console.log(announcement);
+  }
+}
+assessSituation2 (99, `${announcement}`, `${excuse}`);
+assessSituation2 (21, `${announcement}`, `${excuse}`);
+assessSituation2 (3, `${announcement}`, `${excuse}`);
+
+
+
+
 // Declare a new variable - scaryMonster - assigned to an Object with the following key/values
 //   - name (string)
 //   - smell (string)
 //   - weight (integer)
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var address = {
+    number: 4320,
+    street: 'Walnut',
+    state: 'NM',
+    zip: 32094
+}
+
+var scaryMonster = {
+    name:'Ooogaboooga',
+    smell:'broccoli water',
+    weight:398,
+    citiesDestroyed:['Sedona', 'San Francisco', 'Seattle'],
+    luckyNumbers:[3, 4345, 321, 543, 240],
+    address: address
+}
+
+console.log(scaryMonster);
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+
+class SuperHero {
```

Comment:
Awesome work with the content of this class!

Style/syntax wise - check in on indentation.